src/index.tsx

```diff
@@ -141,6 +141,8 @@ serviceWorker.register()
       });
     }
   })
-  .catch((e) => {
-    Sentry.captureException(e);
+  .catch((e: Error) => {
+    if (e) {
```

Comment:
whats this all about? is there a case where its passing something falsey?

Reply:
Yes, on initial load, I get a throw where the error is undefined. Maybe that's just a local thing? Doesn't seem to happen on the Heroku version. I can take it back out.

Reply:
The service worker error doesn't happen locally because SW is disabled outsided production. The SecurityError does happen on the Heroku app. I have put the ignore block back in.